src/nrtk_explorer/library/object_detector.py

```diff
@@ -1,15 +1,11 @@
 import logging
-import operator
 import torch
 import transformers
 
-from functools import reduce
 from typing import Optional
 
 from nrtk_explorer.library import images_manager
 
-Annotations = list[list[tuple[str, dict]]]
```

Comment:
It seems that we can bring this type back

Reply:
Added back a return type annotation.